src/utils/autocomplete-users.js

```diff
@@ -0,0 +1,37 @@
+import * as api from '@/utils/api.js';
+
+export const minSearchLength = 2;
+
+export function userToAutocompleteValue(user) {
+  return {
+    name: user.full_name,
+    details: user.email,
+  };
+}
+
+export function autocompleteValueToUser(value) {
+  return {
+    full_name: value.name,
+    email: value.details,
+  };
+}
+
+export async function* getUsersRaw(query, filterFunction = null) {
```

Comment:
Thought about both options, having `null` seems both faster and more respectful to the "nothing" concept

Reply:
What's up with the "nothing" concept? Here you are defining a default value. If it was "nothing", it would be `undefined`, which is Javascript's default value when you pass "nothing" (don't pass anything) for that argument. A default value is a value to fallback to when "nothing" is given.